Add docs on new cluster UI access/permissions (#1764)
qdrant/landing_page: qdrant-landing/content/documentation/cloud/cluster-access.md

## Cluster UI

> There is the convenient link on the cluster detail page in the Qdrant Cloud Console to access the [Cluster UI](/documentation/web-ui/). You will be automatically authenticated if you have the `read:cluster_data` or `write:cluster_data` permission. If you don't have either of these permissions, you will be prompted with a pop-up to enter an API key to access the cluster.

Reviewer: ```suggestion
You can access your [Cluster UI](/documentation/web-ui/) via the Cluster Details page in the Qdrant Cloud Console. Authentication to a cluster is automatic if your cloud user has the [`read:cluster_data` or `write:cluster_data` permission](/documentation/cloud-rbac/permission-reference/). Without the correct permissions you will be prompted to enter an [API Key](/documentation/cloud/authentication/) to access the cluster.
```
